README.md

```diff
@@ -49,9 +49,9 @@ Learning from another squad and duplicating the code gives many of the same bene
 _Inventing and caring about a thing has a huge hidden cost - you have to build/configure it, maintain it, it spreads focus, broadens competence needs, takes time in maintenance, increases potential for technical debt etc. All in all, it dramatically increases time to value (in a non-transparent way). It also increases time spent on non-critical tasks. Since our most expensive resource is hours spent by our employees, most hours should be spent building the unique stuff only we have - our business logic and user interaction._
 ___
 
-So, if an existing, well designed, maintained software lib or managed service can do 70% of the job, or fits only 70% unless you redefine your problem - you should work hard to reduce your problem to fit into this slot, before you even consider doing it yourself. AWS Lambdas are a good example of something that has significant constraints on runtime, environment and resources, but should almost always be favoured for its ease of use and reliability.
+So, if an existing, well designed, maintained software lib or managed service can do 70% of the job, or fits only 70% unless you redefine your problem - you should work hard to reduce your problem to fit into this slot, before you even consider doing it yourself. AWS Lambdas are a good example of something that has significant constraints on runtime, environment and resources, but should almost always be favoured for its ease of use and reliability. Another example would be support, if that is not your main product, why would you ever try to build an support platform your self, when you have SaaS like Intercom? You shouldn't, these SaaS tools main mission is to solve support the best way possible, poring millions into that goal. Let somebody else build it, and focus on what you need to build to solve the main problems your product tries to solve. 
```

Comment:
*pouring

og så kanskje litt mye solve i siste vers der, hva med:

, and instead focus on building what is strategic to your product